### PR TITLE
Start governance canister integration

### PR DESCRIPTION
**Pull request by hpeebles**
This is an extremely early WIP.

I wanted to make this PR very early just to get some initial thoughts on if I should continue down this path.

### CODE REVIEW

**Comment by bitdivine** [comment]
Hello @hpeebles .  We too (L2 Crypt) need this code so fully support this effort and would be happy to add more API methods and/or help with testing.

How would you like to test this code?  Testing integration is probably the hardest part.  We can test against a testnet and/or a local CI that has identity and ledger canisters running via dfx.  @nmattia  is working on such a setup for the NNS Dapp.  I have yet to see his code but once it is merged we can probably copy the essentials of that test rig to here.

**Review by nmattia — COMMENTED** [review]
Nice!

> @nmattia is working on such a setup for the NNS Dapp.

I haven't looked into other canisters yet, but as soon as I do I'll let you know.

File: src/governance.ts [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+        description: undefined
+      });
+    }
+    if (!knownNeurons.find(n => n.id === BigInt(28))) {
+      knownNeurons.push({
+        id: BigInt(28),
+        name: "Internet Computer Association",
+        description: undefined
+      });
+    }
```
Comment (ielashi) [inline src/governance.ts:88]: As discussed offline, it would be great to move this logic out of this library. Otherwise LGTM!

**Review by ielashi — APPROVED** [review]

File: src/governance.ts (hunk 1 above)
Comment (hpeebles) [reply src/governance.ts]: Done

File: candid/governance.certified.did [hunk 2]
```diff
@@ -0,0 +1,10 @@
+type KnownNeuron = record {
```
[the remark this answers is not in the document]
Comment (hpeebles) [reply candid/governance.certified.did:3]: Done

**Pull request merged**